developer_documentation/contribution_guidelines.Rmd

```diff
@@ -403,7 +401,7 @@ discuss the following questions:
     }
 
     if (!("BioCro" %in% installed.packages()[,"Package"])) {
-      devtools::install_github("ebimodeling/biocro")
+      devtools::install_github("biocro/biocro")
```

Comment:
This might not be the best example anymore, since `devtools::install_github('biocro/biocro')` seems to no longer work due to the git submodule

Reply:
I forgot about this until I'd already done the merge.  I'll make a separate issue and/or pull request to deal with this issue.